library/src/com/sothree/slidinguppanel/SlidingUpPanelLayout.java

```diff
@@ -998,15 +998,17 @@ protected boolean drawChild(Canvas canvas, View child, long drawingTime) {
         if (isSlidingEnabled() && mSlideableView != child) {
             // Clip against the slider; no sense drawing what will immediately be covered,
             // Unless the panel is set to overlay content
-                canvas.getClipBounds(mTmpRect);
-                if (!mOverlayContent) {
-                    if (mIsSlidingUp) {
-                        mTmpRect.bottom = Math.min(mTmpRect.bottom, mSlideableView.getTop());
-                    } else {
-                        mTmpRect.top = Math.max(mTmpRect.top, mSlideableView.getBottom());
-                    }
+            canvas.getClipBounds(mTmpRect);
+            if (!mOverlayContent) {
+                if (mIsSlidingUp) {
+                    mTmpRect.bottom = Math.min(mTmpRect.bottom, mSlideableView.getTop());
+                } else {
+                    mTmpRect.top = Math.max(mTmpRect.top, mSlideableView.getBottom());
                 }
-                canvas.clipRect(mTmpRect);
+            }
+            canvas.clipRect(mTmpRect);
```

Comment:
the behavior is the same, the diff is just hard to read due to whitespace changes: https://github.com/umano/AndroidSlidingUpPanel/pull/330/files?w=1